libsplinter/src/network/peer_manager/interconnect.rs

```diff
@@ -523,14 +523,22 @@ pub mod tests {
 
         let dispatch_shutdown = network_dispatch_loop.shutdown_signaler();
 
+        let mut subscriber = peer_connector.subscribe().expect("Unable to get subscriber");
+
         let peer_ref = peer_connector
             .add_peer_ref("test_peer".to_string(), vec!["test".to_string()])
             .expect("Unable to add peer");
 
         assert_eq!(peer_ref.peer_id, "test_peer");
 
-        // wait to be told to shutdown, timeout after 2 seconds
-        let test_timeout = std::time::Duration::from_secs(2);
+        let notification = subscriber.next().expect("Unable to notification");
```

Comment:
Message should be "Unable to get notification" if the intent was the same as the prior message.

Reply:
fixed